Makefile

```diff
@@ -116,6 +116,19 @@ tensorboard:
 	tensorboard --logdir=$(MODELS) --host=0.0.0.0 &
 	python utils/tb_log_parser.py --prefix=
 
+
+install-opuscleaner:
```

Comment:
Suggestion(docs): Since the Makefile is a developer-facing entry point into the code, it would be nice to provide a short high-level piece of documentation describing what each these actions are. It would also be nice to mention what OpusCleaner is for someone onboarding to the codebase.

> OpusCleaner is a data cleaner for training data. https://github.com/hplt-project/OpusCleaner